extensions/wikia/AffiliateService/crowdin.conf

```diff
@@ -0,0 +1,5 @@
+[crowdin.project]
+projectid = "AffiliateService"
+product_person = "tbd"
```

Comment:
Can we add someone here even if it's cake?

Reply:
Will add `Cake` as the product person here and on the WatchShow crowdin.conf.